lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -546,7 +546,7 @@ func convertContainerToContainerInfo(c *exec.Container) *models.ContainerInfo {
 	}
 
 	if transient {
-		op := trace.NewOperation(context.Background(), "state refresh triggered by a transient data state")
+		op = trace.NewOperation(context.Background(), "state refresh triggered by a transient data state")
```

Comment:
I think we should make this operation a child of the one passed into the outer function, and only use it within this block (as it looks like this operation is supposed to be specifically for the state refresh).

Something like:
```go
 	if transient {
		refreshOp := trace.FromOperation(op, "state refresh triggered by a transient data state")
		c.Refresh(refreshOp)
		container = c.Info()
	}
```

Reply:
done!